### ISSUE TITLE
Add account: "The given url does not seems to work"

### ISSUE THREAD

**Issue by afc54** [dimtion/Shaarlier#65]
Shaarli running on my own freedombox (https://freedombox.org/).  It works fine (as far as I can tell).  I can connect to it from multiple computers (via browser).  Installed shaarlier from F-droid on my phone (galaxy S6, Android 7) and when adding account I get the above message.  I have tried with http, https.  To log in from computers the freedombox shaarli uses user name and password.  I have entered those into the proper (deprecated) shaarlier fields, but it makes no difference.  I have even disabled certificate validation but it makes no difference.  The freedombox uses a self-signed certificate.  Incidentally, the proper spelling for the error message is "The given url does not seem to work" (no S at the end of seem).

**Comment by noccio** [comment]
The same problem with a yunohost installation

**Comment by Pupo1** [comment]
I have the same problem on a Debian 11 installation with Apache server. I have a letsencrypt certificate. I don't know where the log file is, so I can't see anything about the exact error.

**Comment by kpgmr72** [comment]
Shaarlier does not appear to connect correctly with Shaarli version 12. My installation worked fine until tonight when I finished upgrading shaarli from 8.4 to 12. Let me know what logs I can collect and I'll do what I can to help figure this bug/issue out.

The example below was run from the public demo instance of Shaarli.

![Screenshot_20230227_080308_Shaarlier.jpg](https://user-images.githubusercontent.com/5661264/221584439-cf99534f-cad0-4ed7-906a-e65cfad368fd.jpg)

**Comment by jimmyppi** [comment]
It works when using API key instead of username/password, see https://github.com/dimtion/Shaarlier/issues/56

**Comment by Pupo1** [comment]
Does not work for me.

The logs do not show anything special.


Am 26. August 2023 14:00:10 MESZ schrieb jimmyppi ***@***.***>:

**Comment by jimmyppi** [comment]
@Pupo1 so you still get "the given url does not seem to work" when using API key instead of username/password?

**Comment by Pupo1** [comment]
No. I get "the given url is not a compatible Shaarli" but I use exactly the address that I use to enter in my browser, too.

Am 27. August 2023 11:34:04 MESZ schrieb jimmyppi ***@***.***>:
